scripts/systems.py

```diff
@@ -161,16 +224,17 @@
     "Allowed rpns": [8],
     "Multiple rpn in same job": True,
     "GPU": True,
-    "Run command": "srun -u -n {total_ranks} --cpu-bind=core -c {threads_per_rank}",
+    # Based on https://docs.lumi-supercomputer.eu/runjobs/scheduled-jobs/distribution-binding/#gpu-binding
+    "Run command": "srun -u -n {total_ranks} --cpu-bind=mask_cpu:ff000000000000,ff00000000000000,ff0000,ff000000,fe,ff00,ff00000000,ff0000000000 gpu2ranks_slurm",
```

Comment:
Why the 5th is `FE`?

Reply:
From the link in the comment:

```
The LUMI-G compute nodes have the low-noise mode activated. This mode reserve 1 core to the operating system. As a consequence only 63 cores are available to the jobs. Jobs requesting 64 cores/node will never run.
```

So one of the ranks will use one thread less because of that.

Reply:
Ok,
Anyway, shouldn't it be:
`ff00000000000000ff000000000000,ff00000000000000ff00000000000000,ff00000000000000ff0000,ff00000000000000ff000000,fe00000000000000fe,ff00000000000000ff00,ff00000000000000ff00000000,ff00000000000000ff0000000000`?

Reply:
It can be, but need not be since we're not using hyperthreads. Would you prefer the full mask?

Reply:
Yes, IIRC when we tested on daint (long time ago) it was giving better results.

Reply:
Done in https://github.com/eth-cscs/DLA-Future/pull/786/commits/92d50fe9e07ac7a1c42c7d7f7add9af7ab56d855.